app/controllers/api/subcollections/security_groups.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module Subcollections
     module SecurityGroups
       def security_groups_query_resource(object)
-        object.respond_to?(:security_groups) ? object.security_groups : []
+        object.respond_to?(:security_groups) ? object.security_groups : [].freeze
```

Comment:
this .freeze is not needed. can you remove it/repush ?  Thanks.  the frozen literal rubocop is unrelated.